helpers/voting/getVoteMetaData.ts

```diff
@@ -28,19 +25,16 @@ export function getVoteMetaData(
       "No description was found for this UMIP.";
     const umipOrUppUrl = umipDataFromContentful?.umipLink;
     const umipOrUppNumber = getUmipOrUppNumberFromUrl(umipOrUppUrl);
-    const links = makeVoteLinks(transactionHash, umipOrUppNumber, umipOrUppUrl);
```

Comment:
I removed the link building logic from this metadata function because we now use a different async function for getting the transaction hash. this means its better to build links when needed in components, rather than blocking this function which would block the ui